### ISSUE TITLE
experimentalBuildHtml - e2e tests fail in CI when using unminifed ejs files

### ISSUE THREAD

**Issue by yairhaimo** [wix/yoshi#2369]
# 🐛 Bug Report

### This bug is relevant for the `experimentalBuildHtml` flag.
**TLDR** - Yoshi's proxy in CI works only for the unversioned public path url and fails for the unminified ejs files which contain a versioned public path.

### Details
When using the `experimentalBuildHtml`, webpack creates both minified and unminified versions of the ejs files.
In the unminified version, webpack's public path is set to a versioned url from parastorage (the same way as when not using the `experimentalBuildHtml` flag). In the minified version, the public path is set to an unversioned url (#1803).
When building in CI, the public path is set to parastorage (as stated above) but the code is not yet deployed so Yoshi launches puppeteer with a proxy that proxies requests to localhost. Only requests to the public path will be proxied to localhost. The problem is, is that the proxy will redirect only requests that are directed to the **unversioned** url: https://github.com/wix/yoshi/blob/master/packages/jest-yoshi-preset/jest-environment-yoshi-puppeteer/cdnProxy.js#L10
That means that if your server uses the unminified ejs files for e2e tests (which contain a regular version url) then the proxy will not redirect their requests.

### 🤔 Expected Behavior

The CI proxy should support both versioned an unversioned public paths.

### 🔦 Context 

...

### 💁 Possible Solution 

...

### 👣 Steps to Reproduce 

1. Create a yoshi fullstack, app flow, project with `experimentalBuildHtml` enabled
1. Create a template ejs in `src/templates`
1. Load statics from `dist/statics` even in the local env (to load the compiled ejs files):
    ```js
    app.use(
      context.renderer.middleware({
        staticsLocalPathFallback: !isProduction() && './dist/statics',
      })
    );
    ```
1. In your server, return the minified or regular ejs according to whether you're in production or not
1. Tests will fail in CI. Return only the minified version to make the tests pass

### 💻 Code Sample

...

### 🌍 Your Environment

```
  System:
    OS: macOS 10.15.3
    CPU: (12) x64 Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz
  Binaries:
    Node: 12.14.0 - ~/.nvm/versions/node/v12.14.0/bin/node
    Yarn: 1.22.4 - ~/.yarn/bin/yarn
    npm: 6.13.4 - ~/.nvm/versions/node/v12.14.0/bin/npm
    Watchman: 4.7.0 - /usr/local/bin/watchman
  Browsers:
    Chrome: Not Found
    Firefox: 72.0.1
    Safari: 13.0.5
  npmPackages:
    @storybook/core: Not Found
    @storybook/react: Not Found
    babel-preset-yoshi: Not Found
    eslint-config-yoshi: Not Found
    eslint-config-yoshi-base: Not Found
    jest-yoshi-preset: Not Found
    storybook: Not Found
    tslint-config-yoshi: Not Found
    tslint-config-yoshi-base: Not Found
    typescript: Not Found
    webpack: Not Found
    yoshi-config: Not Found
    yoshi-helpers: Not Found
    yoshi-runtime: Not Found

 Yoshi config
{
  "servers": {
    "cdn": {
      "port": 3600
    }
  },
  "splitChunks": {
    "chunks": "all",
    "name": true,
    "maxInitialRequests": 7,
    "minSize": 30000
  },
  "experimentalBuildHtml": true,
  "projectType": "app",
  "cssModules": false,
  "externals": {
    "react": "React",
    "react-dom": "ReactDOM",
    "@wix/fedops-logger": "fedopsLogger"
  },
  "hmr": "auto",
  "entry": {
    "settings": "./components/settings/index.frontend.tsx",
    "view": "./components/view/index.frontend.tsx",
    "embedder": "./components/embedder.ts"
  }
}
```

**Comment by ranyitz** [comment]
Thanks for opening this issue and writing a detailed explanation! I don't think that we'll be able to invest time in solving this in the near future, but we'll definitely fix it before this becomes an official supported feature.
